tests/getMediaUrl.test.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import { ServerlessCallback } from '@twilio-labs/serverless-runtime-types/types';
+import axios from 'axios';
+import { handler as getMediaUrl, Event } from '../functions/getMediaUrl';
+
+import helpers, { MockedResponse } from './helpers';
+
+jest.mock('@tech-matters/serverless-helpers', () => ({
+  ...jest.requireActual('@tech-matters/serverless-helpers'),
+  functionValidator: (handlerFn: any) => handlerFn,
+}));
+
+jest.mock('axios', () => ({
+  request: jest.fn(),
+}));
+
+const mockAxiosRequest = axios.request as jest.MockedFunction<typeof axios.request>;
+
+const baseContext = {
+  getTwilioClient: (): any => ({}),
+  DOMAIN_NAME: 'serverless',
+  ACCOUNT_SID: 'ACCOUNT_SID',
+  AUTH_TOKEN: 'AUTH_TOKEN',
+  PATH: 'PATH',
+  SERVICE_SID: undefined,
+  ENVIRONMENT_SID: undefined,
+};
+
+describe('getMediaUrl', () => {
+  beforeAll(() => {
+    const runtime = new helpers.MockRuntime({});
+    // eslint-disable-next-line no-underscore-dangle
+    helpers.setup({}, runtime);
+  });
+  afterAll(() => {
+    helpers.teardown();
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('Should return status 400 if value is undefined', async () => {
+    const event1: Event = {
+      serviceSid: undefined,
+      mediaSid: 'MIxxxxxxxxxxIOL',
+      request: { cookies: {}, headers: {} },
+    };
+
+    const emptyEvent = {
+      mediaSid: undefined,
+      request: { cookies: {}, headers: {} },
+    };
```

Comment:
The test description says `if value is undefined`. Which value is this? the `serviceSid` or `mediaSid`? Can you specify this in the test description?